modules/rostests/apitests/shell32/She.cpp

```diff
@@ -0,0 +1,87 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for She* functions
```

Comment:
Is it going to serve for other ones (like SheSetCurDrive etc.), or only for the RemoveQuote functions?

Reply:
It's to serve for other ones.